napari/components/viewer_model.py

```diff
@@ -10,12 +10,13 @@
     TYPE_CHECKING,
     Any,
     Dict,
+    Iterator,
     List,
-    Mapping,
     Optional,
     Sequence,
     Set,
     Tuple,
+    Type,
     Union,
 )
```

Comment:
```suggestion
    Optional,
    Sequence,
    Set,
    Tuple,
    Union,
)
```

Reply:
since `from future import __annotations` is used we don't have to stick to 3.8 conventions of typing.

Reply:
for `list[...]`, and `type[...]`, this seem to be true, but `from __future__ import annotation` is not a complete Backport of all the more recent feature and in my experience do not work well. I would prefer to avoid those until we actually drop support for older Python. 

At least using `|` for union is definitely broken.

I'm happy to update type/list, though there as well for consistency across the napari codebase `type[]` (lowercase) is used nowhere, so my guts tell me not to update this one either.